src/index.js

```diff
@@ -0,0 +1,18 @@
+import './style';
+import './lib/fix-pmc';
+import App from './components/app';
+
+export default App;
+
+if (typeof window!=='undefined') {
+	addEventListener('click', e => {
```

Comment:
This should be in a router component. Also we should be excluding external links from this handler.

Reply:
Agreed - I had meant to rip this out, no need for it.

- [ ] remove garbage router code